java/drivers/driver-hazelcast4plus/src/main/java/com/hazelcast/simulator/tests/map/prunability/KeyPojo.java

```diff
@@ -0,0 +1,38 @@
+package com.hazelcast.simulator.tests.map.prunability;
+
+import com.hazelcast.partition.PartitionAware;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+public class KeyPojo implements Serializable, PartitionAware<Integer> {
+    int a;
+    String b;
+    long c;
+
+    public KeyPojo() {
+    }
+
+    public KeyPojo(int a, String b, long c) {
+        this.a = a;
+        this.b = b;
+        this.c = c;
+    }
+
+    public int getA() {
+        return a;
+    }
+
+    public String getB() {
+        return b;
+    }
+
+    public long getC() {
+        return c;
+    }
+
+    @Override
+    public Integer getPartitionKey() {
+        return Objects.hash(a, c);
```

Comment:
does it generate the same partition id as `AttributePartitioningStrategy`? Shouldn't you rather return array of `a` and `c`?

Reply:
I can't guarantee you it generates the same partition id, but it actually doesn't matter. The same approach is used - entries are partitioned by multiple key components. Compared to `AttributePartitioningStrategy `,  distribution may be different.